client/src/PoemBox.jsx

```diff
@@ -44,7 +44,8 @@ class PoemBox extends React.Component {
       url: this.props.url,
       dataType: 'json',
       type: 'POST',
-      data: poem,
+      contentType: 'application/json',
```

Comment:
Yeah, never mind. That might've been from earlier versions of jQuery. Currently, the charset is automatically utf-8.